config/api.yml

```diff
@@ -473,6 +473,29 @@
       :get:
       - :name: read
         :identifier: miq_cloud_networks_view
+  :cloud_volumes:
+    :description: Cloud Volumes
+    :identifier: cloud_volume
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
If it's just as a primary collection then yes, subcollection should go.  Thanks.